core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -592,6 +594,9 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/exploration/StateObjectFactory.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/utilities/StopwatchObjectFactory.js"></script>
 
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/TutorCardDirective.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/SupplementalCardDirective.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/ExplorationStateService.js"></script>
```

Comment:
Should be ExplorationPlayerStateService.js, and the role of the service should be to store data that is needed for the exploration player to work.